sandstone/pattern-video-player/src/App/App.js

```diff
@@ -16,107 +16,91 @@ import css from './App.module.less';
 
 const getVideo = (index) => videos[index];
 
-class AppBase extends Component {
-	static propTypes = {
-		/**
-		 * Assign an alternate panel index to start on.
-		 *
-		 * @type {Number}
-		 * @default 0
-		 * @public
-		 */
-		panelIndex: PropTypes.number,
+const AppBase = ({className, ...rest}) => {
```

Comment:
Since we are deleting these below,

```suggestion
const AppBase = ({className, panelIndex, videoIndex, ...rest}) => {
```

Reply:
If I fix it according to the suggestion, a lint error occurs(19:30  error  Parsing error: Identifier 'panelIndex' has already been declared) and I changed the name of the prop and applied it.